lib/resque_cleaner/server.rb

```diff
@@ -230,7 +230,7 @@ def build_urls
         f: @from,
         t: @to,
         regex: @regex
-      }.map {|key,value| "#{key}=#{URI.encode(value.to_s)}"}.join("&")
+      }.map {|key,value| "#{key}=#{CGI.escape(value.to_s)}"}.join("&")
```

Comment:
There's one more occurrence of this on `lib/resque_cleaner/server/views/_stats.erb:13` that also needs to be updated.